string message
int32 send_counter
int32 get_counter

int16[] set_angle
int16[] set_pressure
int16[] set_current

int16[] get_angle
int16[] get_pressure
char[] brake_pos_limit
bool emergency_stop_enable
int16 motor_control_power
int16 motor_main_power
char plc_control_mode







